pkg/resourcemanager/azuresql/azuresqldb/azuresqldb.go

```diff
@@ -5,9 +5,11 @@ package azuresqldb
 
 import (
 	"context"
+	"fmt"
 	"net/http"
 
 	"github.com/Azure/azure-sdk-for-go/services/preview/sql/mgmt/2015-05-01-preview/sql"
+	sql3 "github.com/Azure/azure-sdk-for-go/services/preview/sql/mgmt/v3.0/sql"
```

Comment:
I need Sql Mgmt v3.0 to accomplish this - so I aliased to something else so that both can co-exist